src/app/static/src/app/components/plots/utils.ts

```diff
@@ -161,6 +162,8 @@ export const formatOutput = function (value: number | string, format: string, sc
 };
 
 const getPlotDataForIndicator = (indicatorMetadata: ChoroplethIndicatorMetadata, plotData: CalibrateDataResponse["data"]) => {
+    // if plotData doesn't have indicator key then we don't filter by it
+    if (!plotData?.at(0)?.hasOwnProperty("indicator")) return plotData;
```

Comment:
When do we not have an indicator?

Reply:
survey data has indicators but art and anc data in the inputs does not have indicator

Reply:
Oh yep sorry being little slow here.